teamcity: failed tests on master: testrace (proposer evaluated kv)/TestStatusLocalLogs

The following tests appear to have failed:

[#81157](https://teamcity.cockroachdb.com/viewLog.html?buildId=81157):

```
--- FAIL: testrace (proposer evaluated kv)/TestStatusLocalLogs (0.003s)
	status_test.go:238: expected log file name servertest.a7fa07147c19.agent.2016-12-05T20_51_41Z.002315.INFO.log to contain ERROR, INFO or WARNING.
	status_test.go:238: expected log file name servertest.a7fa07147c19.agent.2016-12-05T20_51_41Z.002315.WARNING.log to contain ERROR, INFO or WARNING.
	status_test.go:238: expected log file name servertest.a7fa07147c19.agent.2016-12-05T20_51_42Z.002315.ERROR.log to contain ERROR, INFO or WARNING.

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/server/status_test.go
+++ b/pkg/server/status_test.go
@@ -228,8 +228,8 @@ func TestStatusLocalLogs(t *testing.T) {
 	for _, fileInfo := range wrapper.Files {
 		fName := fileInfo.Name
 		found := false
-		for i, name := range []string{"ERROR.log", "INFO.log", "WARNING.log"} {
-			if strings.Contains(wrapper.Files[i].Name, name) {
+		for _, name := range []string{"ERROR.log", "INFO.log", "WARNING.log"} {
+			if strings.Contains(fName, name) {
 				found = true
 				break
 			}

